invenio/modules/workflows/utils.py

```diff
@@ -472,12 +474,12 @@ def get_func_info(func):
     varnames = func.func_code.co_freevars
     if closure:
         for index, arg in enumerate(closure):
-            parameters.append((str(varnames[index]), str(arg.cell_contents)))
+            parameters.append((unicode(varnames[index]), unicode(arg.cell_contents)))
     return {
```

Comment:
@jalavik `unicodifier` can help you here ... `return unicodifier({...})` and recursive magic should happen.